README.md

```diff
@@ -59,6 +59,7 @@ rules.
 - Spotify
 - [SwiftLint](https://github.com/realm/SwiftLint)
 - Tinder
+- Ergatta
```

Comment:
Can you move this to right after `Envoy Mobile? Thanks!

Reply:
I am a little embarrassed I didn't realize it was alphabetical, but done!